prototype_source/prototype_index.rst

```diff
@@ -82,6 +82,13 @@ Prototype features are not available as part of binary distributions like PyPI o
    :link: ../prototype/pt2e_quant_ptq_static.html
    :tags: Quantization          
 
+.. customcarditem::
+   :header: PyTorch 2.0 Export Post Training Static Quantization with X86 Backend through Inductor
+   :card_description: Learn how to use TorchInductor with X86 CPU as backend of Quantization in PyTorch 2.0 Export.
+   :image: ../_static/img/thumbnails/cropped/generic-pytorch-logo.png
+   :link: ../prototype/pt2e_quant_ptq_static_x86_inductor.html
```

Comment:
can you remove `static` from the path here as well

Reply:
Done